documentation/logit-io.md

```diff
@@ -128,8 +128,41 @@ filter {
       # Debug: Comment this line to keep the original object
       remove_field => "[app][payload][params]"
     }
+     mutate {
+            rename => { "[app][payload][status]" => "[http][response][status_code]" }
+          }
+
+     mutate {
+        rename => { "[app][payload][method]" => "[http][request][method]" }
+      }
+
+     mutate {
+        rename => { "[app][payload][format]" => "[http][request][mime_type]" }
```

Comment:
This may be https://www.elastic.co/guide/en/ecs/current/ecs-http.html#field-http-response-mime-type instead. Can you double check?

Reply:
Thanks , your correct  this is reponse mime type.